README.md

```diff
@@ -87,20 +122,34 @@ In the root of the project you can find a `docker-compose.yml` file which provid
 
 You can initialise the schema of the database by running the `schema.sql` which is also provided.
 
+Alternatively, there is a setup script in `bin/db/setup` that will create a database, user and load the schema required for a working development environment. This script assumes the existence of few packages (`toml2json`, `jq`, `postgres`, etc.) made available by the nix-shell but should be usable if they are installed manually.
+
 ## Documentation
 
 You can generate the documentation of the project using
 
-```
+```sh
+# With convenience script
+bin/api/docs
+
+# With Stack
 stack haddock
 ```
 
 ### OpenApi documentation
 
-You can access the OpenAPI documentation just by visiting the `docs` endpoint
+You can access the OpenAPI documentation just by visiting the `docs` endpoint (by default http://localhost:8080/docs)
 
 ## Frontend
 
 This repository contains also a client [Elm](https://elm-lang.org/) application to interact in a human-friendly way with the Tagger api.
 
-You can find more details in [elm/README.md](elm/README.md).
+You can find more details in [elm/README.md](elm/README.md), but there are convenience commands:
+
+```
+# builds the elm project
+bin/frontend/build
+
+# serves, reloading on file change
+bin/frontend/serve
```

Comment:
the scripts provided here are different than the instructions in the `elm\README.md` file. Would be nine to have a common narrative.